apps/analysis/serializers.py

```diff
@@ -420,10 +420,10 @@ class Meta:
             'id',
             'title',
             'team_lead',
-            'project',
             'start_date',
             'end_date',
             'cloned_from',
+            'analysis_pillar',
```

Comment:
Can you confirm this change?

Reply:
Yes, the change to include the `analysis_pillar` field in the fields option within the Meta class of the `AnalysisGqlSerializer` is correct. This adjustment ensures that the `analysis_pillar` field is now included in the serialization process and will resolve the AssertionError related to the missing field.

Reply:
Conversation moved to https://github.com/the-deep/server/pull/1516#discussion_r1697935570